### PR TITLE
format and add deploy password on travis.yml

### PR DESCRIPTION
**Pull request by drgarcia1986**
Related to #66

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,41 @@
...
-    - redis
-    - docker
-
+- redis
+- docker
 deploy:
   provider: pypi
   user: drgarcia1986
-  distributions: "sdist bdist_wheel"
+  distributions: sdist bdist_wheel
```
Comment (jairhenrique) [inline .travis.yml]: if you drop quotations, travis only run `sdist`..

Comment (drgarcia1986) [reply .travis.yml]: I didn't it 😱 
I only ran `travis encrypt` command 🤔

Comment (drgarcia1986) [reply .travis.yml]: Are you sure about it? I can change without problems, I only want to understand

Comment (jairhenrique) [reply .travis.yml]: We fix this issue on https://github.com/luizalabs/django-toolkit/blob/master/.travis.yml#L31

Comment (drgarcia1986) [reply .travis.yml]: Great!

Comment (drgarcia1986) [reply .travis.yml]: @jairhenrique done, could you check again?

**Pull request merged**